src/Manifest/Manifest.router.ts

```diff
@@ -83,6 +92,26 @@ export class ManifestRouter extends Router {
     }
   }
 
+  async getProjectManifestByCoordinates(req: AuthRequest) {
+    const created_location = server.extractFromReq(req, 'land')
+    const eth_address = req.auth.ethAddress
+
+    const project : any = await Project.findOne({
+      created_location,
+      eth_address,
+      is_deleted: false,
+    })
+    if (project) {
+      const body = await new S3Project(project.id).readFileBody(
+        MANIFEST_FILENAME
+      )
+      if (body) {
+        return JSON.parse(body.toString())
+      }
+    }
+    return 'false'
```

Comment:
We could just return a boolean here, right? it'd be correctly handled as we're ultimately returning a JSON.